app/Transformers/TextTransformer.php

```diff
@@ -92,9 +92,9 @@ public function transformForV2($text)
              *   @OA\Xml(name="v2_text_verse"),
              *   @OA\Items(
              *              @OA\Property(property="book_name",         ref="#/components/schemas/Book/properties/name"),
-             *              @OA\Property(property="book_id",           ref="#/components/schemas/Book/properties/id_osis"),
+             *              @OA\Property(property="book_id",           ref="#/components/schemas/Book/properties/name"),
              *              @OA\Property(property="book_order",           ref="#/components/schemas/Book/properties/protestant_order"),
-             *              @OA\Property(property="chapter_id",        ref="#/components/schemas/BibleFile/properties/chapter_start"),
+             *              @OA\Property(property="chapter_id",        ref="#/components/schemas/BibleFile/properties/chapter"),
```

Comment:
BibleFile model schema does not have a chapter property, is chapter_start or chapter_end.

Reply:
noted. I will update the PR with this fix